build-tools/packages/build-cli/src/commands/run/bundleStats.ts

```diff
@@ -5,9 +5,13 @@
 
 import { execSync } from "node:child_process";
 import path from "node:path";
+import { fileURLToPath } from "node:url";
 import { Flags } from "@oclif/core";
 
-import { BaseCommand } from "../../base";
+import { BaseCommand } from "../../base.js";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```

Comment:
This doesn't look off, but I don't think the path using it below is correct.
dangerfile is in `library` and not `lib` and the extension should now be `.cjs` and not `.js`.